src/js/close-button.js

```diff
@@ -6,20 +9,55 @@ import Component from './component';
  * when it is activated.
  *
  * @extends Button
- * @class CloseButton
  */
 class CloseButton extends Button {
 
+  /**
+   * The constructor for CloseButton
+   *
+   * @param {Player} player
+   *        the player to attach this button to
+   *
+   * @param {Object} [options]
+   *        Object of option names and values
+   */
   constructor(player, options) {
     super(player, options);
     this.controlText(options && options.controlText || this.localize('Close'));
   }
 
+  /**
+   * Constructs a string to be used for the component elements
```

Comment:
Does jsdoc not provide any functionality for inheritance? So we would only need to document these in Component and then everything would be linked?